build.gradle

```diff
@@ -18,7 +18,7 @@ android {
     buildToolsVersion '23.0.2'
     defaultConfig {
         applicationId "com.kaytat.simpleprotocolplayer"
-        minSdkVersion 8
+        minSdkVersion 16
```

Comment:
I can't accept this.  I don't want to raise the sdk version.  This pull request is at least two changes.  One for the kbd issue and the *OMG* the network discovery.

Can you split this up?  I'd like to take the kbd fix by itself and we can talk about the discovery by itself.